test/core/support/vector_test.cc

```diff
@@ -18,18 +18,49 @@
 
 #include "src/core/lib/support/vector.h"
 #include <gtest/gtest.h>
+#include "src/core/lib/support/memory.h"
 #include "test/core/util/test_config.h"
 
 namespace grpc_core {
 namespace testing {
 
 TEST(InlinedVectorTest, CreateAndIterate) {
-  InlinedVector<int, 1> v{1, 2, 3};
-  int sum = 0;
-  for (auto i : v) {
-    sum += i;
+  const int kNumElements = 9;
+  InlinedVector<int, 2> v;
+  for (int i = 0; i < kNumElements; ++i) {
+    v.push_back(i);
   }
-  EXPECT_EQ(6, sum);
+  EXPECT_EQ(static_cast<size_t>(kNumElements), v.size());
+  for (int i = 0; i < kNumElements; ++i) {
+    EXPECT_EQ(i, v[i]);
+  }
+}
+
+TEST(InlinedVectorTest, ValuesAreInlined) {
```

Comment:
would it be possible to test whether the values really are inlined, as opposed to placed in heap memory?

Reply:
I'm not sure I see an easy way to do that.  Any suggestions?

Reply:
There are ways of detecting whether a variable lives on the stack vs the heap, but unfortunately they aren't portable. And for a tiny test like this it's not worth doing more sophisticated stuff such as using custom allocators. Nevermind.